Select-Issues-for-Contributing-Using-Labels.md

```diff
@@ -1,42 +1,42 @@
-In this article you can find labels (see the table below) used to categorize the variety of issues submitted to FCC. The labels range from **bugs** to **questions** to **curriculum requests**. For example, if an issue has a ***Wiki*** label, you can create a Wiki article about that issue - after reading the [Wiki Style Guide](Wiki-Style-Guide). 
+In this article you can find labels (see the table below) used to categorize the variety of issues submitted to FCC, along with descriptions (work in progress). The labels range from **bugs** to **questions** to **curriculum requests**. For example, if an issue has a ***Wiki*** label, you can create a wiki article about that issue—after reading the [Wiki Style Guide](Wiki-Style-Guide). 
 
-You can use labels to help you choose which issues to click on, rather than randomly clicking to find one of interest to you. Of course, you can work on and submit PRs for issues without labels. How ever you choose an issue, please be sure to read [Guidelines for Contributing](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md) before submitting any pull requests.
+You can use labels to help you choose which issues to click on, rather than randomly clicking to find one of interest to you. Of course, you can work on and submit PRs for issues without labels. However you choose an issue, please be sure to read the [Guidelines for Contributing](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md) before submitting any pull requests.

```

Comment:
_Well, actually…_

“However” in the original text was being used in its secondary sense of “in whatever way”;* it was just misspelled. More wordily, it was saying, “Whether you choose an issue by using labels or not, RTFM.”

If you add a comma, however, that “However” switches to its most common meaning: “to introduce a statement that contrasts with or seems to contradict something that has been said previously.”* There’s no contradiction with what precedes it, though; “please be sure to read the Guidelines” applies to all issues, regardless of how they are chosen.

\* New Oxford American Dictionary